go-packages/meep-applications/application-store.go

```diff
@@ -0,0 +1,304 @@
+/*
+ * Copyright (c) 2021  InterDigital Communications, Inc
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package applications
+
+import (
+	"errors"
+	"strconv"
+	"sync"
+
+	dkm "github.com/InterDigitalInc/AdvantEDGE/go-packages/meep-data-key-mgr"
+	log "github.com/InterDigitalInc/AdvantEDGE/go-packages/meep-logger"
+	redis "github.com/InterDigitalInc/AdvantEDGE/go-packages/meep-redis"
+)
+
+const redisTable int = 0
+const appMgrKey string = "apps:"
+const (
+	fieldId      string = "id"
+	fieldName    string = "name"
+	fieldMep     string = "mep"
```

Comment:
refer to node instead of mep
Applies to the rest of the file + dependencies to this module